### PR TITLE
Nginx filebeat config

### LINKED ISSUE #809: nginx access logs to elastic
This might require a filebeat sidecar to pick up a separate access log file and push to a new elastic index

### PR DESCRIPTION
**Pull request by BrendanGalloway**
### Description

Enables access logs in nginx ingresses and sets up hints for filebeat nginx module

### Dependencies

NA 

### Breaking Change

- [ ] Yes
- [x] No

### **Links**

Fixes #809

### CODE REVIEW

**Comment by moshloop** [comment]
@BrendanGalloway What does the nginx kibana dashboard look like with this config ?

**Comment by moshloop** [comment]
@BrendanGalloway can you fix linting and semver issues ?

File: test/elastic.yaml [hunk 1]
```diff
@@ -8,9 +8,16 @@ filebeat:
     elasticsearch:
       url: logs-es-http.eck.svc.cluster.local
       user: elastic
-      password: elastic
+      password: s4spxzv9v4p52ndxr7mgvxxp
```
Comment (moshloop) [inline test/elastic.yaml:11]: @BrendanGalloway where does come from?

Comment (BrendanGalloway) [reply test/elastic.yaml:11]: It's from my test cluster.  Will revert

**Pull request merged**